src/address.js

```diff
@@ -18,7 +18,7 @@ function fromBase58Check (address) {
 function toBase58Check (hash, version) {
   typeforce(types.tuple(types.Hash160bit, types.UInt8), arguments)
 
-  var payload = new Buffer(21)
+  var payload = Buffer.allocUnsafe(21)
```

Comment:
I see two variants of this PR: `Buffer.alloc(length)` and `Buffer.allocUnsafe(length)`. Can you choose one?